docker-pipeline.yaml

```diff
@@ -66,6 +66,7 @@ steps:
                "ApiScope": "api1"
              }
           },
+          "DoNotRequireHttps": true,

```

Comment:
This is already done in the step to run the container as an environment variable